TelCo.ColorCoder/ColorCalculation/ColorCalculationForPairNumberFromColor.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace TelCo.ColorCoder
```

Comment:
Hi Prithamvshetty,
First of all I would like to thank you for reviewing my code.

I tried to precise the file name but I thought it might get difficult to understand about the behavior of class that's why I gave like that going forward I'll try to give descriptive names.

Reply:
Descriptive names are indeed helpful. But just be careful that the name should not resemble a line or sentence without space 😄 
Just a thought, can the file name be just ColorForPairNumber ?

Reply:
Done 😄